frontend/src/pages/modelRegistry/screens/RegisteredModelLabels.tsx

```diff
@@ -40,27 +40,35 @@ const RegisteredModelLabels: React.FC<RegisteredModelLabelsProps> = ({
 
   const labelsComponent = (labels: string[], textMaxWidth?: string) =>
     labels.map((label, index) => (
-      <Label color="blue" textMaxWidth={textMaxWidth || '40ch'} key={index}>
+      <Label color="blue" data-testid="label" textMaxWidth={textMaxWidth || '40ch'} key={index}>
         {label}
       </Label>
     ));
 
   const getLabelComponent = (labels: JSX.Element[]) => {
     const labelCount = labels.length;
     if (labelCount) {
-      return labelCount > 7 ? getLabelModal(labelCount) : getLabelPopover(labelCount, labels);
+      return labelCount > 4 ? getLabelModal(labelCount) : getLabelPopover(labelCount, labels);
```

Comment:
nit: This should be a constant and have an explanation, you know this is 4 cause ux told you so, but there's no explanation around this arbitrary number

Reply:
The constant could be at the top of the file